src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+
+  background-image: url("./images/star-active.svg");
+  background-size: 13px 13px;
+
+  height: 13px;
+  width: 13px;
+
+  margin: 3px 7px 3px 3px;
+
```

Comment:
Use correct star container size from design (16x16) and set correct background parameters.